src/main/java/cms/gov/madie/measure/services/QdmTestCaseShiftDatesService.java

```diff
@@ -298,13 +307,24 @@ public List<TestCase> shiftAllTestCaseDates(
     if (CollectionUtils.isEmpty(testCases)) {
       throw new ResourceNotFoundException("TestCases", measureId);
     }
+    StringBuffer testCaseFailures = new StringBuffer();
```

Comment:
Suggest using `StringBuilder` since thread synchronization isn't needed.

Reply:
@jkotanchik-SB I made the change, thanks a lot Joe!